packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1234,6 +1235,14 @@ class CupertinoActionSheetAction extends StatefulWidget {
   /// Destructive buttons have red text.
   final bool isDestructiveAction;
 
+  /// The cursor that will be shown when hovering over the button.
+  ///
+  /// if `null`, cursor will default to:
+  /// - `MouseCursor.click` on web platforms
+  /// - `MouseCursor.defer` on other platforms
+  /// else will use value provided
```

Comment:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar

Reply:
On a smaller note I'm wondering if there is any need for this comment change compared to the previous comment. Maybe the `if null` prefix?

Reply:
reason I changed the comment was mostly cause we are making the mouse cursor null. But I will rever to the below:

/// The cursor that will be shown when hovering over the button.
///
/// If not passed will default to [MouseCursor.defer] and on web will default to [SystemMouseCursors.click].

Reply:
'if not passed' and 'defaults' mean the same thing, so it would be redundant to have the 'if not passed' prefix. However, one can imagine a situation where a user passes `null` as the `mouseCursor`. So technically, in that case, something has been passed. But since we are null coalescing, the rest of the statement would apply.

Maybe a better comment would be `If null, defaults to...` Because that would cover the case where a mouse cursor is not passed and thus there is a default and the case where a null mouse cursor is passed.

Reply:
It seems like a subtle distinction, but it can make a huge difference for a developer trying to debug why a potentially null mouse cursor passed into this widget from somewhere else doesn't change the default to their expected cursor, because in reading the documentation they saw that the defaults are respected only if no mouse cursor is passed in.

Reply:
understood thank you for the comments I have modified it to the following:
/// The cursor that will be shown when hovering over the button.
///
/// if null defaults to [SystemMouseCursors.click] on web else [MouseCursor.defer].